Update control_plane_template.md (#34085)
Azure/azure-rest-api-specs: .github/PULL_REQUEST_TEMPLATE/control_plane_template.md

  [ARM resource provider contract](https://aka.ms/azurerpc) and
  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
  I understand this is required before I can proceed to the diagram Step 2, "ARM API changes review", for this PR.

> - [ ] A [release plan](https://aka.ms/azsdkdocs/release-plans) should have been created, please do so if you have not. A release plan well help guide you through the REST API and SDK creation process. 

Reviewer: ```suggestion
- [ ] A [release plan](https://aka.ms/azsdkdocs/release-plans) has been created. If not lease create one as it will help guide, you through the REST API and SDK creation process. 
```
